components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java

```diff
@@ -6960,6 +6960,7 @@ public void deletePolicyGroup(String applicationId, String policyGroupId) throws
 		String query = "";
 		try {
 	   		conn = APIMgtDBUtil.getConnection();
+            conn.setAutoCommit(false);
```

Comment:
Here inside this function we are using multiple db operations (delete records from child tables) so we are using the same connection. autocommit turned off explicitly to avoid transaction been committed after each execution. Transaction commit is placed end off all db operations. (if fails, set to rollback)